<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Playlist_Viewer</class>
 <widget class="QMainWindow" name="Playlist_Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>294</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Playlist Viewer</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>All Series:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_locations">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>951</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_resume">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume Watching</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollarea_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>951</width>
      <height>181</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>949</width>
       <height>179</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="button_deleteall">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_deleteselected">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Selected</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Currently Playing:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="entry_curplay">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_prev">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_next">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <zorder>scrollarea_list</zorder>
   <zorder>list_locations</zorder>
   <zorder>label_show</zorder>
   <zorder>button_add</zorder>
   <zorder>button_resume</zorder>
   <zorder>button_deleteall</zorder>
   <zorder>button_deleteselected</zorder>
   <zorder>label</zorder>
   <zorder>entry_curplay</zorder>
   <zorder>button_prev</zorder>
   <zorder>button_next</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
